cli/cmd/cluster-apply.go

```diff
@@ -107,6 +110,18 @@ func runClusterApply(cmd *cobra.Command, args []string) {
 		}
 	}
 
+	kubeconfigAbsoutePath, _ := filepath.Abs(kubeconfigPath)
+
+	ctxLogger.Println("Applying admission webhook configuration")
+
+	if err := util.RunAdmissionWebhook(kubeconfigAbsoutePath); err != nil {
+		ctxLogger.Fatalf("Applying admission webhook failed: %v", err)
+	}
+
+	if err := verifyAdmissionWebhook(kubeconfigPath); err != nil {
```

Comment:
Helm already provides a way to wait for the chart to converge, if it's installed with `--wait=true` flag. Perhaps we should make bootkube use it, to ensure that the webhook is in place before we proceed with components installation?

Reply:
Maybe it's an opportunity to look into a mechanism which allows applying arbitrary charts during cluster bootstrap. I think we've learned that it's better not to rely on Bootkube for things other than bootstrapping the k8s control plane.

Reply:
We already have sort of working pattern for that and this task is already broad in my opinion, I'm not sure if we should extend it further.